index.html

```diff
@@ -65,147 +66,158 @@ <h1 class="intro">I'm Ernest Glad to see you!</h1>
+                    <ul class="tonic-section-header">
+                        <li class="canopy">CANOPY</li>
+                        <li><img src="./images/Counter.png" alt="counter" class="counter"/></li>
+                        <li class="back-end-dev">BackEndDev</li>
+                        <li><img src="./images/Counter.png" alt="counter"/></li>
+                        <li class="year">2022</li>
+                    </ul>
+                    <p class="tonic-details">A daily selection of privately personalized reads: no accounts or sign-ups required</p>
+                </div>
+        
+                <ul class="programing-languages">
+                    <li><a href="#">html</a></li>
+                    <li><a href="#">css</a></li>
+                    <li><a href="#">javaScript</a></li>
                 </ul>
-                <p class="tonic-details">A daily selection of privately personalized reads: no accounts or sign-ups required</p>
+                <button type="button">See Project</button>
             </div>
-    
-            <ul class="programing-languages">
-                <li><a href="#">html</a></li>
-                <li><a href="#">css</a></li>
-                <li><a href="#">javaScript</a></li>
-            </ul>
-            <button type="button">See Project</button>
         </div>
 
         <!--Gymfit Section-->
         <div class="Gymfit padding">
             <div class="frontpage-icon">
                 <img src="./images/Gymfit.png" alt="Portfolio" class="frontpage-icon">
             </div>
-            <div class="tonic">
-                <h2>Tonic</h2>
-                <ul class="tonic-section-header">
-                    <li class="canopy">CANOPY</li>
-                    <li><img src="./images/Counter.png" alt="counter" class="counter"/></li>
-                    <li class="back-end-dev">BackEndDev</li>
-                    <li><img src="./images/Counter.png" alt="counter"/></li>
-                    <li class="year">2022</li>
+            <div class="tonic-wrapper">
+                <div class="tonic">
+                    <h2>Tonic</h2>
+                    <ul class="tonic-section-header">
+                        <li class="canopy">CANOPY</li>
+                        <li><img src="./images/Counter.png" alt="counter" class="counter"/></li>
+                        <li class="back-end-dev">BackEndDev</li>
+                        <li><img src="./images/Counter.png" alt="counter"/></li>
+                        <li class="year">2022</li>
+                    </ul>
+                    
+                    <p class="tonic-details">A daily selection of privately personalized reads: no accounts or sign-ups required</p>
+                </div>
+        
+                <ul class="programing-languages">
+                    <li><a href="#">html</a></li>
+                    <li><a href="#">css</a></li>
+                    <li><a href="#">javaScript</a></li>
                 </ul>
```

Comment:
- [ ] The page is divided half of it contains the project's picture and the other half contains the project's description, these switch places with each project (picture is on the left and description on the right for the first project, description on the left and picture on the right for the second project and so on and so forth). So kindly implement it for all the sections.